apps/docs/src/css/custom.css

```
@@ -1,615 +1,807 @@
+  --ifm-color-primary-dark: #0747a6;
+  --ifm-color-primary-darker: #053780;
+  --ifm-color-primary-darkest: #042c66;
+  --ifm-color-primary-light: #1d7afc;
+  --ifm-color-primary-lighter: #4c9aff;
+  --ifm-color-primary-lightest: #b3d4ff;
+  --ifm-background-color: var(--dp-bg-card);
+  --ifm-background-surface-color: var(--dp-bg-board);
+  --ifm-navbar-background-color: rgba(255, 255, 255, 0.96);
+  --ifm-font-family-base: "Inter", "Noto Sans TC", system-ui, sans-serif;
+  --ifm-heading-font-family: "Inter", "Noto Sans TC", system-ui, sans-serif;
+  --ifm-font-family-monospace: "JetBrains Mono", SFMono-Regular, Consolas, monospace;
```

Stylelint (17.11.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)